src/data/faq_de.json

```diff
@@ -266,15 +266,14 @@
                 "id": "notification",
                 "accordion": [
                     {
-                        "title": "[Apple/iOS]: ENErrorDomain-Fehler 13",
+                        "title": "[Apple/iOS]: Fehler 13",
                         "anchor": "ENError13",
                         "active": true,
                         "textblock": [
-                            "<b>AKTUELL</b> 18.12.2020:  <br/>Wenn Sie auf die Corona-Warn-App Version 1.9.1. aktualisiert haben, kann bei einigen Benutzern folgende Fehlermeldung erscheinen: \"Fehler 13: Ihre Risiko-Ermittlung für den heutigen Tag wurde bereits durchgeführt. Bitte versuchen Sie es in 24 Stunden erneut.\"",
-                            "Die Corona-Warn-App funktioniert trotz dieser Meldung korrekt. Sollten in diesem Zusammenhang noch weitere Probleme auftreten, können sie hier Ihr Problem detailliert schildern: <a href='https://github.com/corona-warn-app/cwa-app-ios/issues/1715' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-app-ios#1715</a>",
-                            "<hr/>VERALTETE INFORMATIONEN: <br/>",
-                            "AKTUELL: Wenn Sie auf iOS-Version 13.6. aktualisiert haben, ist dieser Fehler behoben.",
-                            "Der Fehler ENErrorDomain 13 kommt aus dem Exposure Notification System, das von Apple bereitgestellt und von der App genutzt wird. Es handelt sich dabei um eine Information, dass an diesem Tag bereits eine Risikoanalyse durchgeführt worden ist. Bitte starten Sie Ihr iPhone neu, und lassen die Corona-Warn-App für 24 Stunden geschlossen, dann wird erneut eine Risikoanalyse durchgeführt und dieser Fehler sollte nicht mehr angezeigt werden. Bitte deinstallieren Sie die Corona-Warn-App nicht, sie funktioniert weiterhin."
+                            "Der Fehler 13 kommt aus dem Exposure Notification System, das von Apple bereitgestellt und von der App genutzt wird.",
+                            "Es handelt sich dabei um eine Information, dass in den letzten 24 Stunden schon sechs Risiko-Überprüfungen durchgeführt wurden. Apple erlaubt mehr als sechs Risiko-Überprüfungen in 24 Stunden nicht.",

```

Comment:
> Es handelt sich dabei um eine Information, dass in den letzten 24 Stunden schon sechs Risiko-Überprüfungen durchgeführt wurden

"Der Fehler tritt auf falls in den letzten 24 Stunden mehr als 6 Risiko-Überprüfungen durchgeführt wurden"

> Apple erlaubt mehr als sechs Risiko-Überprüfungen in 24 Stunden nicht

"Apple erlaubt nicht mehr als 6 Risiko-Überprüfungen innerhalb von 24 Stunden."